crates/lib/src/jekyll/include_tag.rs

```diff
@@ -13,16 +13,39 @@ use liquid_core::{ParseTag, TagReflection, TagTokenIter};
 #[derive(Debug)]
 struct Include {
     partial: Expression,
+    vars: Vec<(String, Expression)>,
 }
 
 impl Renderable for Include {
     fn render_to(&self, writer: &mut dyn Write, runtime: &mut Runtime<'_>) -> Result<()> {
         let name = self.partial.evaluate(runtime)?.render().to_string();
+
+        let mut varaibles_evaluated = Vec::new();
+        for (id, expr) in &self.vars {
```

Comment:
Any reason you don't insert into the `Object` directly at this point?

Reply:
Fixed that now, orginally was using `runtime` to evaluate it without realising I could just use `scope` to do so.